docs/reference/configuration/configuration.md

```diff
@@ -125,7 +126,7 @@ The order in which overrides are considered is:
 
 #### Defining configuration parameters
 
-The configuration system understands configuration parameters defined in targets, libraries and applications. The configuration parameters are always defined in a JSON object called "config".
+The configuration system understands configuration parameters that targets, libraries and applications define. A JSON object called "config" always defines the configuration parameters.
```

Comment:
Overall I appreciate the preference for active voice, but I find it distracting in these 2 sentences. (Just my opinion. I defer to your judgement as to whether to change it.)